Gemfile.lock

```diff
@@ -539,3 +538,6 @@ DEPENDENCIES
   uglifier (>= 1.3.0)
   warden-github (= 1.4.0)!
   wkhtmltopdf-heroku
+
+BUNDLED WITH
+   1.11.2
```

Comment:
This had been specifically removed to avoid bundler version mismatches - is it cool now?